app/controllers/gobierto_admin/gobierto_plans/plans_controller.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+module GobiertoAdmin
+  module GobiertoPlans
+    class PlansController < GobiertoAdmin::GobiertoPlans::BaseController
+      def new
+        @plan_form = PlanForm.new(site_id: current_site.id)
+        @plan_visibility_levels = plan_visibility_levels
+        @plan_types = find_plan_types
+      end
+
+      def edit
+        @plan = find_plan_by_slug
+        @plan_types = find_plan_types
+        @plan_visibility_levels = plan_visibility_levels
+
+        @plan_form = PlanForm.new(
+          @plan.attributes.except(*ignored_plan_attributes).merge(site_id: current_site.id)
+        )
+      end
+
+      def create
+        @plan_form = PlanForm.new(plan_params.merge(site_id: current_site.id))
+
+        if @plan_form.save
+          track_create_activity
+
+          redirect_to(
+            edit_admin_plans_plan_path(@plan_form.plan.slug),
+            notice: t(".success_html", link: gobierto_plans_plan_preview_url(@plan_form.plan, host: current_site.domain))
+          )
+        else
+          @plan_types = find_plan_types
+          @plan_visibility_levels = plan_visibility_levels
+
+          render :new
+        end
+      end
+
+      def update
+        @plan = find_plan
+
+        @plan_form = PlanForm.new(
+          plan_params.merge(id: params[:id], site_id: current_site.id)
```

Comment:
no need to add the `id` to the `plan_params`

Reply:
But if I don't add it, when I update a plan, it generates a new one

Reply:
You are right, sorry!